sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/MessageUtilsTest.java

```diff
@@ -60,4 +68,33 @@ void convertUUIDToDotNetBytes() {
         // Assert
         assertArrayEquals(dotNetGuidBytes, convertedBytes, "UUID conversion from Java to DotNet failed");
     }
+
+    @ParameterizedTest
+    @MethodSource("calcTotalTimeoutTestData")
```

Comment:
Can we spell out these abbreviations? calc -> calculate.

Reply:
Spelt out